plugins/chat/assets/javascripts/discourse/components/chat-channel.js

```diff
@@ -113,11 +113,10 @@ export default class ChatLivePane extends Component {
     if (this._loadedChannelId !== this.args.channel?.id) {
       this.unsubscribeToUpdates(this._loadedChannelId);
       this.chatChannelPane.selectingMessages = false;
-      this.chatChannelComposer.message =
-        this.args.channel.draft ||
-        ChatMessage.createDraftMessage(this.args.channel, {
-          user: this.currentUser,
-        });
+
+      if (this.args.channel.draft) {
+        this.chatChannelComposer.message = this.args.channel.draft;
+      }
```

Comment:
I changed my mind on this and decided to just initialize this in the setup of the composer if we don't have it yet